ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-ref.yaml

```diff
@@ -0,0 +1,46 @@
+ref:
+  as: openshift-qe-orion
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  env:
+  - name: ORION_REPO
+    default: "https://github.com/cloud-bulldozer/orion.git"
+    documentation: |-
+      Override the e2e version
+  - name: TAG
+    default: "latest"
```

Comment:
Was thinking if we want to have a version number hard-coded here like we do for e2e as of today, but i think for orion it should be fine.